private/bufpkg/bufcheck/buflint/buflint_test.go

```diff
@@ -942,17 +931,104 @@ func TestCommentIgnoresCascadeOff(t *testing.T) {
 
 func TestCommentIgnoresCascadeOn(t *testing.T) {
 	t.Parallel()
+	testLintWithModifiers(t, "comment_ignores_cascade", func(config *bufconfig.Config) {
+		config.Lint.AllowCommentIgnores = true
+	}, nil, "")
+}
+
+func TestValidateRulesTypeDontMatch(t *testing.T) {
+	t.Parallel()
+	testLintWithValidate(t, "validate_rules_types_match_fail",
```

Comment:
The rest of our testdata is in a directory named after the rule - can you stick to this pattern and make sure that all testdata subdirectories are named `name_of_rule` + potentially `_additional_context_needed_for_specific_test`?

Reply:
resolved in [1b2c71c](https://github.com/bufbuild/buf/pull/2377/commits/1b2c71cceeb54702ea58bc69970a1a434fb6410e)